nilearn/plotting/surf_plotting.py

```diff
@@ -1588,10 +1629,11 @@ def plot_surf_roi(surf_mesh,
 
         .. versionadded:: 0.7.1
 
-    alpha : float or 'auto', default='auto'
-        Alpha level of the :term:`mesh` (not the stat_map).
-        If default, alpha will default to 0.5 when no bg_map is passed
-        and to 1 if a bg_map is passed.
+    alpha : float or 'auto' or None, default=None
+        Alpha level of the :term:`mesh` (not surf_data).
+        Will default to ``"auto"`` of ``None`` is passed.
```

Comment:
```suggestion
        Will default to ``"auto"`` if ``None`` is passed.
``` + same comment as above